docs/admin/reference/removing_gpg_passphrase.rst

```diff
@@ -0,0 +1,27 @@
+Removing the Passphrase from a GPG Key
```

Comment:
These instructions need better context - they should be performed on dom0 and in a temporary GPG homedir that gets nuked after they're complete. I had some notes in a recent related support issue if that helps.